invenio_app_rdm/utils/files.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio App RDM is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Files utilities."""
+
+from __future__ import absolute_import, print_function
+
+from datetime import datetime
+from urllib.parse import quote
+
+import sqlalchemy as sa
+from flask import current_app
+from flask_mail import Message
+from invenio_files_rest.models import FileInstance
+from invenio_rdm_records.records.models import RDMDraftMetadata, RDMRecordMetadata
+
+
+def checksum_verification_files_query():
+    """Return a FileInstance query taking into account file URI prefixes."""
+    files = FileInstance.query
+    uri_prefixes = current_app.config.get(
+        "FILES_REST_CHECKSUM_VERIFICATION_URI_PREFIXES"
+    )
+    if uri_prefixes:
+        files = files.filter(
+            sa.or_(*[FileInstance.uri.startswith(p) for p in uri_prefixes])
+        )
+    return files
+
+
+URLS = {
+    "deposit_html": "{base}/uploads/{id}",
```

Comment:
I tried to use a config here for record/deposit routes, however did not find an adequate and noted that these paths are hardcoded elsewhere.

Let me know if there is a more consistent/robust way to retrieve paths (e.g.`/uploads` and `/records`).